Simplenote/src/main/java/com/automattic/simplenote/utils/NoteUtils.java

```diff
@@ -53,4 +64,48 @@ public static String getWordCount(String content) {
         int words = (content.trim().length() == 0) ? 0 : content.trim().split("([\\W]+)").length;
         return NumberFormat.getInstance().format(words);
     }
+
+    public static void showDialogDeletePermanently(final Activity activity, final Note note) {
+        new AlertDialog.Builder(new ContextThemeWrapper(activity, R.style.Dialog))
+            .setTitle(R.string.delete_dialog_title)
+            .setMessage(R.string.delete_dialog_message)
+            .setNegativeButton(R.string.no, null)
+            .setPositiveButton(
+                R.string.yes,
+                new DialogInterface.OnClickListener() {
+                    public void onClick(DialogInterface dialog, int whichButton) {
+                        if (note != null) {
+                            note.delete();
+
+                            // Show empty placeholder for large devices in landscape.
+                            if (activity instanceof NotesActivity) {
+                                ((NotesActivity) activity).showDetailPlaceholder();
+                            // Close editor for small devices and large devices in portrait.
+                            } else if (activity instanceof NoteEditorActivity) {
+                                ((NoteEditorActivity) activity).finish();
+                            }
+                        } else {
+                            showDialogErrorDelete(activity);
+                        }
+                    }
+                }
+            )
+            .show();
+    }
+
+    private static void showDialogErrorDelete(Context context) {
```

Comment:
Idea (💡): I am seeing that this is a duplicate of the `TagDialogFragment.showDialogErrorRename()' method, isn't it? Maybe consider removing the duplication by reusing this one. Wdyt?

Reply:
I noticed that too.  The [`delete_dialog_error_message_email`](https://github.com/Automattic/simplenote-android/blob/259e51e5ac495ed4bd0fcd313620197c4c48ce76/Simplenote/src/main/res/values/strings.xml#L45) string resource is also a duplicate of [`rename_tag_message_email`](https://github.com/Automattic/simplenote-android/blob/259e51e5ac495ed4bd0fcd313620197c4c48ce76/Simplenote/src/main/res/values/strings.xml#L21).  I considered consolidating the methods and resources, but I didn't want to creep too far out of scope of this pull request.  I already crept a little by changing the **_Empty Trash_** dialog message.  I'll combine the methods and resources in a subsequent pull request.